### PR TITLE
Replaced bootstrap-tour with introJs tour library in Story Activity

### PR DESCRIPTION
**Pull request by abhishek-sultaniya**
Fixes: #1090 
I have replaced bootstrap-tour with introJs tour library in Story Activity.All the suggestions,improvements and modifications are welcome.I followed all the steps as described in [here](https://github.com/llaske/sugarizer/pull/1083#issuecomment-1167648209).

### CODE REVIEW

File: activities/Story.activity/js/components/SugarTutorial.js [hunk 1]
```diff
@@ -1,62 +1,35 @@
...
-		}
-	}
+// Tutorial component based on  introjs tour
+Vue.component("sugar-tutorial", {
+  data: function () {
+    return {
+      l10n: {
+        stringTutoPrev: "Prev",
+        stringTutoNext: "Next",
+        stringTutoEnd: "End",
```
Comment (disha1202) [inline activities/Story.activity/js/components/SugarTutorial.js:8]: stringTutoEnd can be removed as we're no longer using it.

File: activities/Story.activity/js/components/SugarTutorial.js [hunk 2]
```diff
@@ -1,62 +1,35 @@
...
+    let vm = this;
+    if (this.$root.$refs.SugarL10n) {
+      this.$root.$refs.SugarL10n.$on("localized", function () {
+        vm.$root.$refs.SugarL10n.localize(vm.l10n);
+      });
+    }
+  },
+  methods: {
+    show: function (steps) {
+      introJs()
```
Comment (disha1202) [inline activities/Story.activity/js/components/SugarTutorial.js:22]: Filter the steps before starting the tutorial.
https://github.com/llaske/sugarizer/pull/1083/files#diff-43815ec1886368b47450dfe256cc92d8594fd7c643a362644fc9aeeef5fbc519R22

**Review by disha1202 — CHANGES_REQUESTED** [review]
This is great work 👏🏻 I have suggested some minor changes which you can incorporate.

**Comment by abhishek-sultaniya** [comment]
Thanks, @disha1202 , Now corrected these two suggestions in the recent commit:

Removed stringTutoEnd SugarTutorial.js
Now the steps are filtered before starting the tutorial.

**Comment by llaske** [comment]
@abhishek-sultaniya I think some styles are missing. The size of the header is too small and I think the text spacing is not the same.

image

**Comment by abhishek-sultaniya** [comment]
Hey @llaske ,In recent commit I have changed the style.After the commit it looks as follows:-
image

**Comment by llaske** [comment]
It's better but I wonder about the close button position like I've mentioned [here](https://github.com/llaske/sugarizer/pull/1099#issuecomment-1236190370).
Plus why it will be need to specify `line-height` in this activity and not elsewhere?

**Comment by disha1202** [comment]
> It's better but I wonder about the close button position like I've mentioned [here](https://github.com/llaske/sugarizer/pull/1099#issuecomment-1236190370).
 Plus why it will be need to specify `line-height` in this activity and not elsewhere?
 
 I faced the same issue with header, this is also because of different versions of IntroJs
 https://github.com/usablica/intro.js/pull/1708/files#diff-61919747715934800867ffcb198d7e8a6450cb3fbd4c8902cd3afbbc807a6d60R203

**Comment by disha1202** [comment]
> It's better but I wonder about the close button position like I've mentioned [here](https://github.com/llaske/sugarizer/pull/1099#issuecomment-1236190370).
 Plus why it will be need to specify `line-height` in this activity and not elsewhere?
 
 I faced the same issue with header, this is also because of different version of IntroJs
 https://github.com/usablica/intro.js/pull/1708/files#diff-61919747715934800867ffcb198d7e8a6450cb3fbd4c8902cd3afbbc807a6d60R203

**Comment by abhishek-sultaniya** [comment]
I think it is due to different versions of introjs.What should I do,handle it using custom css or try some previous version of introjs?

**Comment by llaske** [comment]
Try to use the same version than the one currently use in Sugarizer to see if it's solve the issue.

**Comment by abhishek-sultaniya** [comment]
> Try to use the same version than the one currently use in Sugarizer to see if it's solve the issue.

Hello @llaske ,in recent commits I have used the version of introjs used in SUgarizer and removed the line-height from activity.css.
Please review it.

**Comment by llaske** [comment]
It works. Thanks.

**Pull request merged**